eml/api/api.js

```diff
@@ -257,27 +196,14 @@ export const fetchCertificates = async (userId) => {
   }
 };
 
-
-// Call to backend to see if online
-export const checkBackendOnline = async () => {
-  let response;
-  try {
-    const res = await axios.get(url + '/api/utility/online/', {timeout: 1000});
-    response = res.data;
-  } catch {
-    response = false;
-  }
-  return response;
```

Comment:
why delete?

Reply:
This functionality is now done by the NetworkService functions, and so, this call is not in use any more

Reply:
TODO: Remember to delete /api/utility/online/ route from backend